### ISSUE TITLE
commit message is inconsistent with code in fifth commit.

### ISSUE THREAD

**Issue by reed9527** [rui314/chibicc#93]
commit: 25b4b85b887c643e337a9fbcd1b0220b413952bf

commit message: Add ==, !=, <= and >= operators

actually, the code implements ==, !=, < and <= operators

**Comment by Arnab42444** [comment]
No. As much as I am concerned the token LT is used for both < and >, and LE is used for both <=, >=.
